artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/impl/PagingStoreImpl.java

```diff
@@ -700,7 +720,7 @@ public boolean checkMemory(final Runnable runWhenAvailable) {
 
    @Override
    public void addSize(final int size) {
-      boolean globalFull = pagingManager.addSize(size).isGlobalFull();
+      boolean globalFull = !ignoreGlobalMaxSize && pagingManager.addSize(size).isGlobalFull();
```

Comment:
@clebertsuconic @mtaylor 
I'm not adding anything to the pagingManager to not trigger the other addresses policies: it makes sense?

Reply:
This would affect the reported size though, which for someone operating the broker is important it returns a correct and valid value.

Reply:
I agree: indeed it was one of my concern on it.
On one side to not impact other addresses and trigger paging due to management messages and on the other side to maintain consistency of the metrics related to address utilization.
Obviously just one of the paths can be chosen: I take your point and I agree to maintain correct metrics instead :+1: